GettingStartedTutorial.md

```diff
@@ -216,6 +214,24 @@ To submit credentials, use Postman (or similar, based on your local configuratio
 ]
 ```
 
+For example:
+
+```bash
+curl -H 'Content-Type: application/json' -X 'POST' http://localhost:5000/issue-credential -d @-
+<paste contents of the above file here>
+<CTRL-D>
```

Comment:
The common way to do this in bash is to end the curl line with `<< EOF`  and instead of putting <CTRL-D>, you put `EOF`.

So there... 

I'll still mark this is as approved and you can either merge as is or change.

Reply:
Cool thanks for the hint.  I'll merge the PR and update later ...